src/modules/stores/aws.js

```diff
@@ -2,14 +2,15 @@ const { aws: awsEnv } = require("../../../loadenv");
 
 const logger = require("../logger");
 // Load the AWS-SDK and s3
-const AWS = require("aws-sdk");
-AWS.config.update({
-  region: "ap-northeast-2",
-  signatureVersion: "v4",
-});
+const {getSignedUrl} = require("@aws-sdk/s3-request-presigner")
+const {PutObjectCommand,S3} = require("@aws-sdk/client-s3")
```

Comment:
지난번에 말씀해주신 것처럼, prettier가 동작하지 않은 것 같아요!
+ 오 이번에 CI 단계에 prettier 검사를 넣어볼까요?
```suggestion
const { getSignedUrl } = require("@aws-sdk/s3-request-presigner");
const { PutObjectCommand, S3 } = require("@aws-sdk/client-s3");
```

Reply:
오 너무좋아요..! 아니면 git hook으로 하나 걸어놔도 괜찮을 것 같아요!